modules/autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/jsbridge/response/AutofillDataResponses.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.jsbridge.response
+
+import com.duckduckgo.autofill.domain.javascript.JavascriptCredentials
+
+data class ContainingCredentials(
```

Comment:
Same here too for documentation. Would be good to link into something in the Autofill to know why these data is structured this way.

Reply:
Deferring; tracked in https://app.asana.com/0/0/1202592915624853/f